File: Java/Product.java
/* Saya Listia Ningrum NIM 2008084 mengerjakan Latihan 2 dalam mata
	kuliah DPBO untuk keberkahanNya maka saya tidak akan melakukan kecurangan 
	seperti yang telah dispesifikasikan. Aamiin */

class Product 
{
	// deklarasi atribut
	private String idProduct;
	private int price;
	
	// constructor
	public Product(String idProduct, int price){
		this.idProduct = idProduct;
		this.price = price;
	}
	public Product(){}
	
	// method set & get id product
	public void setIdProduct(String idProduct){
		this.idProduct = idProduct;
	}
	public String getIdProduct(){
		return this.idProduct;
	}
		
	// method set & get price
	public void setPrice(int price){
		this.price = price;
	}
	public int getPrice(){
		return this.price;
	}
}

File: PHP/Product.php
<?php

/* Saya Listia Ningrum NIM 2008084 mengerjakan Latihan 2 dalam mata
	kuliah DPBO untuk keberkahanNya maka saya tidak akan melakukan kecurangan 
	seperti yang telah dispesifikasikan. Aamiin */

class Product 
{
	// deklarasi atribut
	private $idProduct = "";
	private $price = 0;
	
	// constructor
	public function __construct($idProduct = "", $price = 0){
		$this -> idProduct = $idProduct;
		$this -> price = $price;
	}
	
	// method set & get id product
	public function setIdProduct($idProduct){
		$this -> idProduct = $idProduct;
	}
	public function getIdProduct(){
		return $this -> idProduct;
	}
		
	// method set & get price
	public function setPrice($price){
		$this -> price = $price;
	}
	public function getPrice(){
		return $this -> price;
	}
}

?>
